src/include/nfs/ui.rb

```diff
@@ -324,13 +324,13 @@ def GetFstabEntry(fstab_ent, existing)
             # Translators: 1st part of error message
             error_msg = _("No NFS server has been found on your network.")
 
-            if SuSEFirewall.GetStartService
+            if firewalld.running?
               # Translators: 2nd part of error message (1st one is 'No nfs servers have been found ...)
               error_msg = Ops.add(
                 error_msg,
                 _(
                   "\n" \
-                    "This could be caused by a running SuSEfirewall2,\n" \
+                    "This could be caused by a running Firewall,\n" \
```

Comment:
NP: I'd use `firewall` (with lowercase `f`) here in the meaning of a generic firewall.